gui/default/assets/lang/lang-en.json

```diff
@@ -415,10 +437,11 @@
     "Take me back": "Take me back",
     "The GUI address is overridden by startup options. Changes here will not take effect while the override is in place.": "The GUI address is overridden by startup options. Changes here will not take effect while the override is in place.",
     "The Syncthing Authors": "The Syncthing Authors",
-    "The Syncthing admin interface is configured to allow remote access without a password.": "The Syncthing admin interface is configured to allow remote access without a password.",
+    "The Syncthing admin interface is configured to allow remote access without authentication.": "The Syncthing admin interface is configured to allow remote access without authentication.",
     "The aggregated statistics are publicly available at the URL below.": "The aggregated statistics are publicly available at the URL below.",
     "The cleanup interval cannot be blank.": "The cleanup interval cannot be blank.",
     "The configuration has been saved but not activated. Syncthing must restart to activate the new configuration.": "The configuration has been saved but not activated. Syncthing must restart to activate the new configuration.",
+    "The credentials below were created with a different setting of \"Webauthn Rp Id\" in advanced settings. They cannot be used without restoring this setting to the value shown in the \"RP ID\" column below. Note that doing so would make any credentials above ineligible instead.": "The credentials below were created with a different setting of \"Webauthn Rp Id\" in advanced settings. They cannot be used without restoring this setting to the value shown in the \"RP ID\" column below. Note that doing so would make any credentials above ineligible instead.",
```

Comment:
Better not call it a "column", because that will invalidate translations if we choose a different layout later.  Maybe just `... shown under "RP ID" below.`

Reply:
Fixed in f5c31ef6bb202092c16c29f8d450964a6568dd05.